mysite/w6/beanstalk.py

```diff
@@ -24,17 +24,21 @@ def __get_beanstalk_data(raw):
 
 
 def section_beanstalk():
-    beanstalk_bought = "Gold Food Beanstalk" in session_data.account.jade_emporium_purchases
-    upgrade_bought = "Supersized Gold Beanstacking" in session_data.account.jade_emporium_purchases
+    beanstalk_bought = (
+        "Gold Food Beanstalk" in session_data.account.jade_emporium_purchases
+    )
+    upgrade_bought = (
+        "Supersized Gold Beanstacking" in session_data.account.jade_emporium_purchases
```

Comment:
Not a fan of this- IMO worse readability to spread this assignment onto 3 lines

Reply:
`black` auto did this for the file, I'm down to change them back, I'm not a huge fan either when its just doing a single statement like that

Reply:
I ended up changing this code anyway. Resolving.